src/sensai/vector_model.py

```diff
@@ -39,7 +39,7 @@ def getPredictedVariableNames(self) -> list:
         pass
 
 
-# TODO Is this obsolete?
+# TODO Do we need this class?
```

Comment:
The only reason to keep it could be for type annotations in the evaluation module. The base evaluator classes only need the predictor and fittable model interfaces. Which means if we at some point want fittable models that are not vector models (or we want to allow our advanced users to build them), the base evaluator classes can be used for them as well

Do you think it's enough of a reason? It probably doesn't hurt to keep it, I can't see any downsides

Reply:
Yeah, let's keep it.